modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -16,7 +16,8 @@ const defaultProps = {
   uniqueIdProperty: {type: 'string', value: ''},
   highlightedFeatureId: null,
   loaders: [MVTLoader],
-  binary: false
+  binary: false,
+  triangulate: false
```

Comment:
Is this necessary? Should the layer just request triangulation if `binary: true`?

Any new API must be added to the layer's documentation.

Reply:
@alasarr what do you think? I think it also makes sense to always request triangulation if `binary: true`. If we don't then it'll be slower and the holes are broken.

Reply:
After checking with @alasarr I believe we are all in agreement that it is simpler and logical to always request the triangulation when `binary: true`. As such I've removed this option